sysutils/smart/Makefile

```
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		smart
-PLUGIN_VERSION=		1.5
+PLUGIN_VERSION=		2.0
```

```suggestion
PLUGIN_VERSION=		2.0.d
```

Set this to a "development" version to be able to finish the conversion